tests/test_sevm.py

```diff
@@ -362,3 +364,85 @@ def test_large_memory_offset(sevm: SEVM, solver, storage):
 
     [output_ex] = exs
     assert isinstance(output_ex.context.output.error, OutOfGasError)
+
+
+def test_jump_into_push_data(sevm, solver, storage):
+    hexcode = binascii.unhexlify("60055663015b000000")  # Convert hex string to bytes
+    exec = mk_ex(hexcode, sevm, solver, storage, caller, this)
+
+    execs = list(sevm.run(exec))
+
+    # Ensure execution halted due to an invalid jump
+    assert len(execs) == 1  # Only one execution path should exist
+    assert execs[0].context.output.error is not None  # Ensure an error occurred
+    assert isinstance(
+        execs[0].context.output.error, InvalidJumpDestError
+    )  # Check correct error type
+
+
+def test_jumpi_false_no_error(sevm, solver, storage):
+    hexcode = binascii.unhexlify(
+        "6000600657005b00"
+    )  # PUSH1 0x03; PUSH1 0x00; JUMPI; STOP
```

Comment:
comment is wrong here too, I see PUSH1 00, PUSH1 06, ...

might be interesting to end with INVALID rather than STOP